README.md

```diff
@@ -1,6 +1,6 @@
 ![Terraform Provider Iterative](https://static.iterative.ai/img/cml/banner-terraform.png)
 
-# Terraform Iterative provider
+# Iterative Provider [![](https://img.shields.io/badge/-documentation-5c4ee5?logo=terraform))](https://registry.terraform.io/providers/iterative/iterative/latest/docs)
```

Comment:
```suggestion
# Iterative Provider [![](https://img.shields.io/badge/-documentation-5c4ee5?logo=terraform))][tpi]

[tpi]: https://registry.terraform.io/providers/iterative/iterative/latest/docs

```

Reply:
There may be too many `)`s in there too?

Reply:
I even wonder if the ref link works for images i.e. 

```md
![][tf-logo]

[tf-logo]: https://img.shields.io/badge/-documentation-5c4ee5?logo=terraform
```

Reply:
I am very strongly against ref links unless they are reused `#error-unused-abstraction`.